pub/index.html

```diff
@@ -0,0 +1,86 @@
+                    <blockquote style="display:block; margin-top:32px;">
+                        <p style="font-size:17px; font-family:'PT serif' serif; color:#bbb; line-height:26px; font-style:italic; border-left:solid 2px #e8e8e8; padding-left:15px;"><em>Maddie and Kolby</em></p>
+                    </blockquote>
+                    <p style="font-size:17px; font-family:'PT serif' serif; color:#666666; line-height:26px; margin-top:32px;">This is my boyfriend Kolby, I love him with all my heart. He makes me want to be a better person and conquer all my fears. He puts up with a lot from me but we really balance each other well. He's my little Arkansas Ginger Bird!</p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/kolby1.jpg"></p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/kolby2.jpg"></p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/kolby3.jpg"></p>
+                </article>
+                <article style="border-bottom: solid 2px #e8e8e8; padding-bottom:38px;">
+                    <h1 style="color:#39474e; font-size:40px; font-family:'Open-sans' sans-serif; font-weight:bolder; margin-top:32px;">My Family</h1>
+                    <blockquote style="display:block; margin-top:32px;">
+                        <p style="font-size:17px; font-family:'PT serif' serif; color:#bbb; line-height:26px; font-style:italic; border-left:solid 2px #e8e8e8; padding-left:15px;">
+                            <em>Best family a girl could ask for</em>
+                        </p>
+                    </blockquote>
+                    <p style="font-size:17px; font-family:'PT serif' serif; color:#666666; line-height:26px; margin-top:32px;">My family is so entirely important to me. I am a daddy's girl through and through. He has given me so much guidence and direction in life and every opportunity in thw world to succeed. I will forever be greatful to him for all the opportunites I have had. My mom and I are total opposites but it works so wonderfully. She is like my other half and when we are together, we bring out the best in each other. I have never felt so comfortable with someone in my life. I love them to both to pieces.</p>
+                    <p style="font-size:17px; font-family:'PT serif' serif; color:#666666; line-height:26px;">Erin is like a sister to me. She is my rock and I don't know what I would do without her. She has been there for some very hard time in my life and she constantly picks me up. She is my best friend for now and for always.</p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/family1.jpg"></p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/family2.jpg"></p>
```

Comment:
Better wrapped in a `<figure>` than a `<p>` tag, as the latter is reserved for paragraphs of _text_...